### PR TITLE
Changed input parameter pvtfqdn to node_id

### PR DESCRIPTION
**Pull request by akash2144**
Signed-off-by: Akash Dudhane <akash.a.dudhane@seagate.com>

## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    Change input parameter pvtfqdn to node_id
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes/No
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Previously we are considering nodeid as pvfqdn (node_name). we need to change it to nodeid.
  </code>
</pre>
## Solution
<pre>
  <code>
    Changed input parameter from pvtfqdn to nodeid
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Unit Testing details here...
  </code>
</pre>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: ha/core/controllers/pcs/node_controller.py [hunk 1]
```diff
@@ -170,12 +170,21 @@ def check_cluster_feasibility(self, node_id: str) -> dict:
...
             Check whether the cluster is going to be offline after node with node_id is stopped.
         Args:
-            node_id (str): Private fqdn define in conf store.
+            node_id (str): Node ID from cluster nodes.
         Returns:
             Dictionary : {"status": "", "msg":""}
         """
-        # Raise exception if node_id is not valid
-        self._is_node_in_cluster(node_id=node_id)
+        # Get the node_name (pvtfqdn) fron nodeid
```
Comment (ajay-paratmandali) [inline ha/core/controllers/pcs/node_controller.py:177]:
```
        nodeid_dict = self._confstore.get(f"{const.PVTFQDN_TO_NODEID_KEY}")
        node_name = ""
        for key, nodeid in nodeid_dict.items():
            if nodeid == node_id:
                node_name = key.split('/')[-1]
        if node_name == "":
            raise ClusterManagerError(f"Failed to get node_name for nodeid: {node_id} from conf_store")
```
repetition of code seen
use method

node_id = get_exact_nodeid(node_id)

Comment (akash2144) [reply ha/core/controllers/pcs/node_controller.py:177]: Added def _get_node_name(self, node_id: str) -> str:

File: ha/core/controllers/pcs/node_controller.py [hunk 2]
```diff
@@ -273,34 +275,35 @@ def stop(self, node_id: str, timeout: int= -1, **op_kwargs) -> dict:
...
-            if self.heal_resource(node_id):
+            if self.heal_resource(node_name):
                 time.sleep(const.BASE_WAIT_TIME)
             try:
-                Log.info(f"Please Wait, trying to stop node: {node_id}")
-                self._execute.run_cmd(const.PCS_STOP_NODE.replace("<node>", node_id)
-                        .replace("<seconds>", str(timeout)))
-                Log.info(f"Executed node stop for {node_id}, Waiting to stop resource")
+                Log.info(f"Please Wait, trying to stop node: {node_name}")
+                self._execute.run_cmd(f"pcs cluster stop {node_name}")
```
Comment (indrajitzagade) [inline ha/core/controllers/pcs/node_controller.py:299]: Please add a TODO here to add timeout value and use PCS_STOP_NODE from const.py

Comment (akash2144) [reply ha/core/controllers/pcs/node_controller.py:300]: Added TODO

**Review by ajay-paratmandali — APPROVED** [review]

**Review by indrajitzagade — APPROVED** [review]

**Pull request merged**